### DISCUSSION TITLE
Allow users to filter what appears in their "Recent" tab on web

### DISCUSSION THREAD

**Discussion by SecreOsu** [ppy/osu#22058] [Ideas]
Would like to have an ability to filter things based on type of action. As a mapper/player I'd like to be able to not have my recent top 1000 scores show in my recent tab (like shown in screenshot) but instead only my mapping updates as I want to be able to have a more streamlined feed of my maps. Being able to filter out things such as medals, gameplay or map submissions would be a nice QOL feature.
![chrome_sFjgKiYN8D](https://user-images.githubusercontent.com/80791374/211124044-0813cc80-7dc0-46a8-b5a6-17a124633a18.png)

**Comment by ch4tie** [comment]
Agree, an option to filter or at least have separate tabs for modding and playing would be nice.

**Comment by frenzibyte** [comment]
Needs a feature request over at https://github.com/ppy/osu-web first.
